packages/tsconfig-shared.json

```diff
@@ -1,6 +1,16 @@
 // This file is used by the tsconfig.json files in each package.
 /* Visit https://aka.ms/tsconfig to read more about this file */
 {
+    "exclude": [
+        "dist",
+        "**/*.test.*",
+        "**/__tests__/**",
```

Comment:
If _all_ of our test (and story and cypress) files lived in `__tests__` then we'd only need to specify the dir. But we have a few .test.ts files outside.